recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  skip: true  # [win]
```

Comment:
We may or may not need some sort of skip depending on what C++ features you require.